src/switcher.js

```diff
@@ -167,10 +281,10 @@ class Switcher extends EventEmitter {
         try {
             var socket = await this._connect(this.SWITCHER_PORT, this.switcher_ip);
             socket.on('error', (error) => {
-                this.log.debug('gloabal error event:', error);
+                this.log('gloabal error event:', error);

```

Comment:
the log function is basically a logger (I used the one I receive from homebridge) so your changes will break the homebridge plugin.